tests/gold_tests/traffic_ctl/remap_inc/remap_inc.test.py

```diff
@@ -0,0 +1,78 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import os
+Test.Summary = '''
+Test traffic_ctl config reload with remap.config .include directive
+'''
+
+Test.ContinueOnFail = False
+
+Test.Setup.Copy("wait_reload.sh")
+
+# Define ATS and configure
+ts = Test.MakeATSProcess("ts", command="traffic_manager", enable_cache=False)
+
+ts.Disk.File(ts.Variables.CONFIGDIR + "/test.inc", id="test_cfg", typename="ats:config")
+ts.Disk.test_cfg.AddLine(
+    "map http://example.two/ http://yada.com/ " +
```

Comment:
yaba.com ?

Reply:
https://www.phrases.org.uk/meanings/yada-yada.html

Reply:
It's just a place-holder for a URL that is not really used.